jobs/deactivateReleaseDatePassedLicences.ts

```diff
@@ -3,10 +3,10 @@
  * Do appinsights first to instrument the logger
  */
 import 'reflect-metadata'
-import { initialiseAppInsights, buildAppInsightsClient, flush } from '../server/utils/azureAppInsights'
+import { initialiseAppInsights, flush } from '../server/utils/azureAppInsights'
+import applicationInfo from '../server/applicationInfo'
 
-initialiseAppInsights()
-buildAppInsightsClient('create-and-vary-a-licence-deactivate-release-date-passed-licences-job')
+initialiseAppInsights(applicationInfo('create-and-vary-a-licence-deactivate-release-date-passed-licences-job'))
```

Comment:
simplified this set up a bit - the buildAppInsightsClient was really "configure default client that is used for everything" and we didn't actually need a reference to it, so merged the two.